go/ql/src/Security/CWE-347/MissingJwtSignatureCheck.qhelp

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE qhelp PUBLIC "-//Semmle//qhelp//EN" "qhelp.dtd">
+<qhelp>
+     <overview>
+          <p>Applications decoding a JSON Web Token (JWT) may be vulnerable when the
+               signature is not correctly verified in the process.</p>
+     </overview>
+     <recommendation>
+     <p>Always verify the signature by using the appropriate methods depending on the JWT library,
+          or use a library that verifies it by default.</p>
+     </recommendation>
+     <example>
+          <p>The following example shows a case where a JWT is parsed without verifying the
+               signature.</p>
+          <sample src="MissingJwtSignatureCheckBad.go" />
+          <p>In the example below, the appropriate function for parsing a JWT
+               and verifying its signature is used.</p>
+          <sample src="MissingJwtSignatureCheckGood.go" />
+     </example>
+     <references>
+          <li>
```

Comment:
? Is an empty LI necessary? Can/should we add a least one citation?

Reply:
🤦‍♂️ Yes, the references were missing, sorry. Here they are: https://github.com/github/codeql/pull/15585/commits/3c40c0f65ba3cc6af555230a14befdd57832ea38.